recipes/pcl/all/conanfile.py

```diff
@@ -4,14 +4,13 @@
 from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
 from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rmdir, rm
 from conan.tools.gnu import PkgConfigDeps
-from conan.tools.microsoft import is_msvc, is_msvc_static_runtime
```

Comment:
You have removed is_msvc_static_runtime but it is used at line 205.

Reply:
I saw the linter error, but it's not used anywhere in the recipe, though. Definitely not on line 205.

Reply:
I think it was a display bug because is_msvc_static_runtime appears in https://github.com/conan-io/conan-center-index/pull/18389/files#diff-6fa762467605014c7cc8570474e6c5c69c760445d7645fee95dbe517ccd10376R498 at line 498.
I'll restore this import.